tools/Makefile

```diff
@@ -138,9 +124,10 @@ installopt::
 	  "$(INSTALL_LIBDIR)"
 
 # To help building mixed-mode libraries (OCaml + C)
+ocamlmklib_cmos = config.cmo build_path_prefix_map.cmo misc.cmo ocamlmklib.cmo
```

Comment:
`ocamlmklib_objects`?

Reply:
> Overall I think this is a good idea - there are a few things which definitely need tweaking before it's equivalent.
> 
> I would also make the variables more consistent - e.g. merge `CAMLDEP_OBJ`/`CAMLDEP_IMPORTS` and `CSLPROF`/`CSLPROF_IMPORTS`, given that each recipe has to repeat the objects. Seems sensible that `tool` has `tools_objects := ...`.
> 
> That said, the removal of macros could go further if you have `tool_imports` containing just `.cma` files and `tools_objects` containing just `.cmi` & `.cmo` files then `byte2native` could go, with `$(tool_imports:.cma=.cmxa) $(tool_objects:.cmo=.cmx)` in its place.

I've removed the redundant variables but kept `byte2native` - I think it's clear and useful, it's a pure function with no weird eval / side-effects / quoting.

Reply:
Agreed - up to you if you want to make the names themselves consistent between tools:

- s/stripdebug/STRIPDEBUG
- s/primreq/PRIMREQ
- s/OBJINFO/OCAMLOBJINFO
- s/ocamlmklib_cmos/OCAMLMKLIB
- s/ocamlcp_cmos/OCAMLCP
- s/CSLPROF/OCAMLPROF
- s/CAMLDEP/OCAMLDEP

Reply:
Done. I think the CAMLDEP -> OCAMLDEP change fixed a bug.